client/post-editor/editor-ground-control/index.jsx

```diff
@@ -153,13 +154,7 @@ export class EditorGroundControl extends PureComponent {
 		return buttonLabels[ primaryButtonState ];
 	}
 
-	shouldShowStatusLabel() {
-		const { isSaving, post } = this.props;
-
-		return isSaving || ( post && post.ID && ! postUtils.isPublished( post ) );
-	}
-
-	isSaveAvailable() {
+	isSaveEnabled() {
```

Comment:
Looks like tests are [expecting](https://github.com/Automattic/wp-calypso/blob/0b3df411b195f66d65ece4794332cc79b1f40986/client/post-editor/editor-ground-control/test/index.jsx#L67-L109) `isSaveAvailable`

> TypeError: tree.isSaveAvailable is not a function

Reply:
Fixed up! Thanks for spotting this @jblz!